### DISCUSSION TITLE
Welcome to mono Discussions!

### DISCUSSION THREAD

**Discussion by zxstim** [bifrost-io/developers#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PaulElisha** [comment]
hey, what ideas are top on your mind?

**Reply by zxstim** [reply]
Check out ideas here (https://github.com/bifrost-io/mono/tree/main/builder-program#ideas)

see what you can whip up on Base. Use the testnet kit here (https://github.com/buildstationorg/bifrost-testnetkit)

**Reply by PaulElisha** [reply]
alright, thank you.
